public/app/features/dashboard-scene/scene/layout-default/DefaultGridLayoutManager.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint @grafana/no-untranslated-strings: "error" */
```

Comment:
Probably this should be removed to avoid ignoring this file out of this task. WDYT?